package.json

```diff
@@ -47,6 +47,7 @@
     "rlp": "^2.1.0",
     "rpc-websockets": "^4.1.1",
     "shelljs": "^0.8.2",
+    "tronweb": "^2.5.3",
```

Comment:
Heh, I wonder how much bloat this adds, might have to restructure loom-js at some point so the Tron stuff can be installed as an addon... but no need to worry about that now.